base/src/main/java/com/smartdevicelink/managers/lifecycle/BaseSystemCapabilityManager.java

```diff
@@ -266,81 +272,90 @@ void parseRAIResponse(RegisterAppInterfaceResponse response) {
+                                if (systemCapabilityResponse.getSuccess() && SystemCapabilityType.DISPLAYS.equals(systemCapability.getSystemCapabilityType())) {
+                                    shouldConvertDeprecatedDisplayCapabilities = false; // Successfully got DISPLAYS data. No conversion needed anymore
+                                    List<DisplayCapability> newCapabilities = (List<DisplayCapability>) systemCapability.getCapabilityForType(SystemCapabilityType.DISPLAYS);
+                                    updateCachedDisplayCapabilityList(newCapabilities);
+                                }
+                                break;
+                        }
+                    } else if (RPCMessage.KEY_NOTIFICATION.equals(message.getMessageType())) {
+                        switch (message.getFunctionID()) {
 							case ON_HMI_STATUS:
 								OnHMIStatus onHMIStatus = (OnHMIStatus) message;
 								if (onHMIStatus.getWindowID() != null && onHMIStatus.getWindowID() != PredefinedWindows.DEFAULT_WINDOW.getValue()) {
 									return;
 								}
 								currentHMILevel = onHMIStatus.getHmiLevel();
 								break;
-							case ON_SYSTEM_CAPABILITY_UPDATED:
-								OnSystemCapabilityUpdated onSystemCapabilityUpdated = (OnSystemCapabilityUpdated) message;
-								if (onSystemCapabilityUpdated.getSystemCapability() != null) {
-									SystemCapability systemCapability = onSystemCapabilityUpdated.getSystemCapability();
-									SystemCapabilityType systemCapabilityType = systemCapability.getSystemCapabilityType();
-									Object capability = systemCapability.getCapabilityForType(systemCapabilityType);
-									if (cachedSystemCapabilities.containsKey(systemCapabilityType)) { //The capability already exists
-										switch (systemCapabilityType) {
-											case APP_SERVICES:
-												// App services only updates what was changed so we need
-												// to update the capability rather than override it
-												AppServicesCapabilities appServicesCapabilities = (AppServicesCapabilities) capability;
-												if (capability != null) {
-													List<AppServiceCapability> appServicesCapabilitiesList = appServicesCapabilities.getAppServices();
-													AppServicesCapabilities cachedAppServicesCapabilities = (AppServicesCapabilities) cachedSystemCapabilities.get(systemCapabilityType);
-													//Update the cached app services
-													if (cachedAppServicesCapabilities != null) {
-														cachedAppServicesCapabilities.updateAppServices(appServicesCapabilitiesList);
-													}
-													//Set the new capability object to the updated cached capabilities
-													capability = cachedAppServicesCapabilities;
-												}
-												break;
-											case DISPLAYS:
-												shouldConvertDeprecatedDisplayCapabilities = false; // Successfully got DISPLAYS data. No conversion needed anymore
+                            case ON_SYSTEM_CAPABILITY_UPDATED:
```

Comment:
On line 311, the alignment of `case ON_SYSTEM_CAPABILITY_UPDATED` was altered . It (and all of its code) needs to be on the same level as the case above it (`case ON_HMI_STATUS`).

Reply:
> On line 311, the alignment of `case ON_SYSTEM_CAPABILITY_UPDATED` was altered . It (and all of its code) needs to be on the same level as the case above it (`case ON_HMI_STATUS`).

@kostyaBoss I still see the alignment issue with `case ON_SYSTEM_CAPABILITY_UPDATED` on line 305.  It (and all of its code) needs to be on the same level as the case above it (`case ON_HMI_STATUS`).